server/public/model/outgoing_oauth_connection.go

```diff
@@ -157,3 +160,23 @@ func (oaf *OutgoingOAuthConnectionGetConnectionsFilter) SetDefaults() {
 		oaf.Limit = defaultGetConnectionsLimit
 	}
 }
+
+// ToURLValues converts the filter to url.Values
+func (oaf *OutgoingOAuthConnectionGetConnectionsFilter) ToURLValues() url.Values {
+	v := url.Values{}
+	v.Set("limit", fmt.Sprintf("%d", oaf.Limit))
```

Comment:
Would you expect to add the parameters regardless of whether they are set?

Reply:
Totally right, thanks for the catch.